src/blocks/affiliate/shared/helpers.js

```diff
@@ -0,0 +1 @@
+export const refreshAffiliateWidgets = () => { window.organic?.cmd?.push(window.organic?.apps?.affiliate?.processPage?.()); };
```

Comment:
> This will call processPage right away and push the result into organic.cmd array
🤦 Doh! I caught this bug at the end of the day ... I probably should have waited until my brain was a little more rested. Thanks for the catch lol.